csrc/executor.cpp

```diff
@@ -1673,6 +1673,14 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
     expr_eval.bind(inputs[i], *args[i]);
   }
 
+  const bool measure_kernel_time = measure_kernel_time_ ||
+      isDebugDumpEnabled(DebugDumpOption::EffectiveBandwidth) ||
+      isDebugDumpEnabled(DebugDumpOption::PerfDebugVerbose);
+
+  if (measure_kernel_time) {
+    inputBytesProcessed(args);
+  }
+
```

Comment:
I needed to move up where the input bytes were measured because the `args` variable captures all of a kernel's args in nvFuser that includes outputs if I wait to look at the `args` farther down.  Therefore, our effective bandwidth measurement was previously double counting outputs.